src/SourceBuild/tarball/content/repos/xliff-tasks.proj

```diff
@@ -2,21 +2,15 @@
   <Import Project="Sdk.props" Sdk="Microsoft.NET.Sdk" />
 
   <PropertyGroup>
-    <BuildCommandArgs>pack</BuildCommandArgs>
-    <BuildCommandArgs>$(BuildCommandArgs) $(ProjectDirectory)src\XliffTasks\XliffTasks.csproj</BuildCommandArgs>
-    <BuildCommandArgs>$(BuildCommandArgs) /v:$(LogVerbosity)</BuildCommandArgs>
-    <BuildCommandArgs>$(BuildCommandArgs) /flp:Verbosity=Diag</BuildCommandArgs>
-    <BuildCommandArgs>$(BuildCommandArgs) /bl</BuildCommandArgs>
-    <BuildCommandArgs>$(BuildCommandArgs) /p:TreatWarningsAsErrors=false</BuildCommandArgs>
-    <BuildCommandArgs>$(BuildCommandArgs) $(RedirectRepoOutputToLog)</BuildCommandArgs>
-
-    <BuildCommand>$(DotnetToolCommand) $(BuildCommandArgs)</BuildCommand>
+    <BuildCommand>$(StandardSourceBuildCommand) $(StandardSourceBuildArgs)</BuildCommand>
 
     <RepoApiImplemented>false</RepoApiImplemented>
     <DependencyVersionInputRepoApiImplemented>true</DependencyVersionInputRepoApiImplemented>
     <DeterministicBuildOptOut>true</DeterministicBuildOptOut>
     <IsToolingProject>true</IsToolingProject>
 
+    <ShippingPackagesOutput>$(ProjectDirectory)artifacts/packages/$(Configuration)/Shipping/</ShippingPackagesOutput>
```

Comment:
```suggestion
    <OutputPlacementRepoApiImplemented>false</OutputPlacementRepoApiImplemented>
```

Reply:
I have removed `ShippingPackagesOutput`. Isn't `OutputPlacementRepoApiImplemented` implied by `RepoApiImplemented` (which is already set to `false`)?

Reply:
You are correct.

Reply:
Thanks for the review!